Kentico.Kontent.Delivery.Abstractions/Models/RichText/RichTextContentConverter.cs

```diff
@@ -5,9 +5,9 @@
 
 using Newtonsoft.Json.Linq;
 using AngleSharp.Parser.Html;
-using Kentico.Kontent.Delivery.ContentLinks;
+using Kentico.Kontent.Delivery.Abstractions.ContentLinks;
 
-namespace Kentico.Kontent.Delivery
+namespace Kentico.Kontent.Delivery.Abstractions
 {
     internal class RichTextContentConverter : IPropertyValueConverter
```

Comment:
this one doesn't belong to abstractions, imo. it contains plenty of functionality. i'd limit the abstractions to models and interfaces only.

it might even make the abstraction thinner by not having to reference anglesharp in the packagereference

Reply:
I moved this class to the Deliver library and delete the necessary library.